helm/artifactory/.gitignore

```diff
@@ -1 +1,2 @@
 charts/
+Chart.lock
```

Comment:
artifactory-ref-impl/deploy
/install.sh  CHART_VERSION=1.3.0-beta.2-develop
change to 
CHART_VERSION=1.3.0-develop